### PR TITLE
[Minor] improvement(docs): Optimized Trino related docs.

### PR DESCRIPTION
**Pull request by yuqi1129**
### What changes were proposed in this pull request?

Optimize the docs of Gravitino Trino connector. 

### Why are the changes needed?

Better user experience. 

### Does this PR introduce _any_ user-facing change?

N/A

### How was this patch tested?

N/A

### CODE REVIEW

**Comment by yuqi1129** [comment]
You may need to add the link to issue #1137.

File: docs/trino-connector/installation.md [hunk 1]
```diff
@@ -134,5 +129,5 @@ system
 test.jdbc-mysql
 ```
 
-The catalog named 'test.jdbc-mysql' is your created catalog by gravitino server. 
-You can use it to access the mysql database like other Trino catalogs.
\ No newline at end of file
+The catalog named 'test.jdbc-mysql' is the catalog that you created by gravitino server, and
```
Comment (jerryshao) [inline docs/trino-connector/installation.md:132]: Remove the trailing white space.

**Review by diqiu50 — DISMISSED** [review]

File: docs/trino-connector/trino-connector.md [hunk 2]
```diff
@@ -7,11 +7,13 @@ This software is licensed under the Apache License version 2."
...
 
 Trino can manage and access data using the Trino connector provided by `Gravitino`, commonly referred to as the `Gravitino connector`.
-After configuring the Gravitino connector in Trino, it can automatically load catalog metadata from Gravitino, allowing users to directly access these catalogs in Trino.
+After configuring the Gravitino connector in Trino, Trino can automatically load catalog metadata from Gravitino, allowing users to directly access these catalogs in Trino.
 Once integrated with Gravitino, Trino can operate on all Gravitino data without requiring additional configuration.
 
-When metadata such as catalogs, schemas, or tables change in Gravitino, Trino can also update itself through Gravitino.
-
+:::node
+Once metadata such as catalogs, schemas, or tables are changed in Gravitino, Trino can update itself through Gravitino immediately, this process usually takes
```
Comment (jerryshao) [inline docs/trino-connector/trino-connector.md:14]: Actually you need to wait for 3-10 seconds, so it is not proper to say "immediately" here.

Comment (yuqi1129) [reply docs/trino-connector/trino-connector.md:14]: OK

File: docs/trino-connector/installation.md [hunk 3]
```diff
@@ -67,30 +66,29 @@ Now you can see the Gravitino connector directory in the plugin directory.
...
-The `gravitino.uri` defines the connection information about Gravitino server. Make sure your container can access the Gravitino server.
+- The `gravitino.name` defines which Gravitino connector is used. It must be `gravitino`.
+- The `gravitino.metalake` defines which metalake are used. It should exist in the Gravitino server.
+- The `gravitino.uri` defines the connection information about Gravitino server. Make sure your container can access the Gravitino server.
 
-If you don't have the `test` metalake. You can create a new metalake named `test`. 
+Full configurations for Gravitino connector can be seen [here](configuration.md)
 
-Create a new metalake named `test` by the following command.
+If you haven't created the metalake named `test`, you can use the following command to create it.
```
Comment (jerryshao) [inline docs/trino-connector/installation.md:85]: Also here.

File: docs/trino-connector/catalog-postgresql.md [hunk 4]
```diff
@@ -43,10 +43,10 @@ PostgreSQL's tables and schemas cannot support properties.
 
 ## Basic usage examples
 
-First, you need to create a metalake and catalog in Gravitino.
-For example, create a new metalake named `test` and create a new catalog named `postgresql_test` using the `jdbc-postgresql` provider.
-And configure the Metalake `test` into the `Graviton connector`.
-For More information about the PostgreSQL catalog, please refer to [PostgreSQL catalog](../jdbc-postgresql-catalog.md).
+You need to do the following steps before you can use the PostgreSQL catalog in Trino through Gravitino.
+
+- Create a metalake and catalog in Gravitino. Assuming that the metalake name is `test` and the catalog name is `postgresql_test`, then you can use
```
Comment (jerryshao) [inline docs/trino-connector/catalog-postgresql.md:48]: ditto

**Comment by yuqi1129** [comment]
All comments have been resolved.

**Review by jerryshao — APPROVED** [review]

**Pull request merged**